challenges/array-magic.md

```
@@ -0,0 +1,121 @@
+# Programming Challenge: [Array Magic with Cards][array-magic]
```

Markdownlint

Missing link or image reference definition: "array-magic"
Reference links and images should use a label that is defined

(MD052, reference-links-images)